server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/LeaderboardService.java

```diff
@@ -27,10 +28,10 @@ public List<LeaderboardEntry> createLeaderboard() {
         logger.info("Found " + users.size() + " users");
```

Comment:
We should also show users that are in the development team but did not do any reviewing

Reply:
On the preview deployment it they are visible 🤔 I guess I was mistaken